chapters/connectors.tex

```diff
@@ -107,9 +107,13 @@ \subsection{Expandable Connectors}\label{expandable-connectors}
 This applies to both variables of simple types, and variables of structured types.
 
 Then connections containing expandable connectors are elaborated:
-\begin{itemize}
+\begin{enumerate}
+\item
+  If a connect-statement references a potentially present component (either as the complete argument or as part of the argument) it will be marked as being present and will be treated as declared below.
```

Comment:
My first interpretation of "will be treated as declared below" was that it would be declared somewhere below. Perhaps changing the order would avoid this confusion:
```suggestion
  If a connect-statement references a potentially present component (either as the complete argument or as part of the argument) it will be marked as being present and will be treated below as declared.
```

Reply:
Ok, seems good.